index.html

```diff
@@ -1,8 +1,55 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Mashup</title>
-  </head>
-  <body>
-  </body>
-</html>
+<head>
+	<title>Mashup</title>
+
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js"></script>
+	<script type="text/javascript" src="mashup.js"></script>
+	<link rel="stylesheet" type="text/css" href="mashup.css">
+	<link href='https://fonts.googleapis.com/css?family=Gloria+Hallelujah' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+	<h1>Mashup Project: Mad Libs</h1>
+
+	<div id="wrapper">
+
+		<div id="firstPic" class="pic">
+			<form class="form" onsubmit="return false;">
+				<input type="text" placeholder="LOCATION"/>
+				<input class="submitForm" type="submit" value="OK" onclick="instaSearch(this);"/>
```

Comment:
How could you avoid including inline JS here?

https://en.wikipedia.org/wiki/Unobtrusive_JavaScript